examples/fleet.yaml

```diff
@@ -50,6 +50,17 @@ spec:
       maxSurge: 25%
       # the amount to decrements GameServers by. Defaults to 25%
       maxUnavailable: 25%
+  # [Stage:Alpha]
+  # [FeatureFlag:FleetAllocationOverflow]
+  # Labels and/or Annotations to apply to GameServers when the number of Allocated GameServers drops below
```

Comment:
Should this be "Labels and/or Annotations to apply to GameServers when the number of the desired replicas on the underlying `GameServerSet` drops below the number of Allocated GameServers"?

Reply:
You are right - I've got it backwards.